tests/crypto/custom-rodata.ld

```diff
@@ -0,0 +1,254 @@
+. = ALIGN(4);
+__start_test_vector_aes_mac_data = .;
+KEEP(*(test_vector_aes_mac_data))
+__stop_test_vector_aes_mac_data = .;
```

Comment:
I'm not convinced we should put the test vectors in named sections.

Could you elaborate on why we need/should do this?

Reply:
This scheme was adopted from the nrf5 source.

Could @frkv chime in here?

Reply:
Discussed offline. named sections are necessary to integrate with host-side tooling.